lib/ci/tacos-apply

```diff
@@ -4,6 +4,15 @@ export TERRAGRUNT_INCLUDE_MODULE_PREFIX=false
 cmd=(tf-lock terragrunt-noninteractive apply --auto-approve)
 log=apply-log.hcl
 
+if [ "$GITHUB_PR_APPROVALS" = "0" ]; then
+    gha-step-summary <<EOF
+### TACOS Apply: $TF_ROOT_MODULE
+
+PR not approved, so apply disallowed.
+EOF
+    exit 1
+fi
```

Comment:
Prefer `[[` over `[`, as long as we're in bash. It's a builtin so slightly faster (avoids fork overhead), but more importantly it has a few less pitfalls than `[`.

```suggestion
if ! [[ "$GITHUB_PR_APPROVERS" ]]; then
    gha-step-summary <<EOF
### TACOS Apply: $TF_ROOT_MODULE
Production changes must be peer-reviewed.
Please try again after code review.

Apply request refused: PR has no approvals
    exit 1
EOF
fi